example-secrets/docker-build-secrets.yml.example

```diff
@@ -0,0 +1,2 @@
+docker-registry-ci: eu.gcr.io/census-rm-ci
+docker-registry-gcr: eu.gcr.io/census-rm-gcr
```

Comment:
This should be census-gcr-rm

Reply:
Also, perhaps time to rename this file to something like `docker-build-config.yml` since it should no longer contain any secrets.